central/notifier/policycleaner/clean_policy.go

```diff
@@ -0,0 +1,30 @@
+package policycleaner
+
+import "github.com/stackrox/rox/central/detection"
+
+// PolicyCleaner removes notifier from policies.
+type PolicyCleaner struct {
+	buildTimePolicies  detection.PolicySet
+	deployTimePolicies detection.PolicySet
+	runTimePolicies    detection.PolicySet
+}
+
+// DeleteNotifierFromPolicies removes notifier from policies.
+func (s *PolicyCleaner) DeleteNotifierFromPolicies(notifierID string) error {
```

Comment:
Either `p` or `c`, `s` definitely feels out of place here.
```suggestion
func (p *PolicyCleaner) DeleteNotifierFromPolicies(notifierID string) error {
```